pkg/openthread/contrib/netdev/openthread_netdev2.c

```diff
@@ -0,0 +1,183 @@
+/*
+ * Copyright (C) 2017 Fundaci�n Inria Chile
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @{
+ *
+ * @file
+ * @author  José Ignacio Alamos <jialamos@uc.cl>
+ */
+
+#include "ot.h"
+#include "msg.h"
+#include "openthread/platform/alarm.h"
+#include "openthread/platform/uart.h"
+#include "openthread/cli.h"
+#include "openthread/instance.h"
+#include "openthread/tasklet.h"
+#include <assert.h>
```

Comment:
This should be the first include: put system includes first, then local includes